java/lib/build.gradle

```diff
@@ -0,0 +1,85 @@
+apply plugin: 'maven-publish'
+apply plugin: 'com.android.library'
+apply plugin: 'org.mozilla.rust-android-gradle.rust-android'
+
+android {
+    ndkVersion '28.0.12433566'
```

Comment:
Latest version is `28.0.12674087`

Reply:
Nice, I'll update it. At the time this PR was created, `28.0.12433566` was the latest version :disguised_face:

Reply:
> At the time this PR was created, `28.0.12433566` was the latest version 🥸

I see the sarcasm, but let’s maintain a professional tone moving forward. Thank you.

Reply:
Oh sorry @tareknaser, it was not sarcasm, I was just stating that I didn't know a new version was released. Sorry if you felt that way. It was actually nice that you let me know my local NDK is outdated